scripts/create_and_deploy_recipe.sh

```diff
@@ -18,8 +18,8 @@
 
 set -e
 
-if [[ "${S3_UPLOAD_ID}" = "" || "${S3_UPLOAD_SECRET}" = "" || "${S3_UPLOAD_BUCKET}" = "" ]]; then
-    echo "You need to export S3_UPLOAD_ID, S3_UPLOAD_SECRECT and S3_UPLOAD_BUCKET for this to work"
+if [[ "${AWS_ACCESS_KEY_ID}" = "" || "${AWS_SECRET_ACCESS_KEY}" = "" ]]; then
```

Comment:
Restore requirement for specifying S3_UPLOAD_BUCKET.

Reply:
Restored the requirement for S3_UPLOAD_BUCKET.